### ISSUE TITLE
During any rebalancing action through thunderhub, red box in the UI pops up says "ExpectedFsToRebalance"

### ISSUE THREAD

**Issue by beeforbacon** [apotdevin/thunderhub#200]
**Describe the problem/bug**
During any rebalancing action through thunderhub, red box in the UI pops up says "ExpectedFsToRebalance. The exact error in logs is:

```
2020-12-29 15:23:22 info [THUB]: Rebalance Params: { out_channels: [ [length]: 0 ], avoid: [ [length]: 0 ] }
2020-12-29 15:23:22 error [THUB]: [ 400, 'ExpectedFsToRebalance', [length]: 2 ]
```

**Your environment**
* Version of ThunderHub: 12.2
* Deployment method: manual nothing special. all integrated on one machine.

**To Reproduce**
1. Go to rebalance
2. Click on advanced, then rebalance (or any specific rebalancing action
4. See error ExpectedFsToRebalance
 

**Additional context**
This suddenly started happening. I have BOS installed and it works fine. Perhaps I have to set the bos account in some way for thunderhub to recognize, but I found no guide on how to do this (ex. on bos I run "bos rebalance --node=node name" 

edit: This issue is still ongoing-- unable to use the Advanced routing tool within THUB still. 
2021-01-15 00:02:10 info [THUB]: Rebalance Params: { out_channels: [ [length]: 0 ], avoid: [ [length]: 0 ] }
2021-01-15 00:02:10 error [THUB]: [ 400, 'ExpectedFsToRebalance', [length]: 2 ]

**Comment by apotdevin** [comment]
Thanks for opening this issue! Fixed with 3c8fb65975a8e1f65099c1177a500c90426503b4 and will be in the next release

**Comment by apotdevin** [comment]
Is now in version 0.12.6
